### PR TITLE
[move] Upgrade to newest state of Move step #2

### PR DESCRIPTION
**Pull request by wrwg**
### Description

Upgrades our repo to the new Move conventions of lower-case module names. 

- Module names were hardcoded in the code about everywhere and in 1/2 dozen different ways. 
- TransactionBuilderGenerator did not work after the renaming, so deactivated it and for now checked in the hand-massaged builders from the previous version.

### Test Plan

Existing tests.

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2032)

### CODE REVIEW

**Review by zekun000 — COMMENTED** [review]
this is insane amount of work, only one thing I want to clarify before approve.

how do the new generated_txn_builder generated? do we manually copy the generated files from the target dir or?

**Comment by wrwg** [comment]
> this is insane amount of work, only one thing I want to clarify before approve.
> 
> how do the new generated_txn_builder generated? do we manually copy the generated files from the target dir or?

The transaction generator doesn't work any longer. At least it didn't when I tried at the beginning of the refactoring. For now its a copy of the generated code before this change, then updated by hand to new naming. We need to fix the builder but not as part of this PR.

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 1]
```diff
@@ -93,7 +91,7 @@ module AptosFramework::Account {
         writeset_epilogue_name: vector<u8>,
         currency_code_required: bool,
     ) {
-        SystemAddresses::assert_aptos_framework(account);
+        assert!(signer::address_of(account) == @aptos_framework, errors::requires_address(ENOT_APTOS_FRAMEWORK));
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/account.move:94]: Why not use system_addresses:: assert_aptos_framework ?

**Review by movekevin — APPROVED** [review]

**Review by areshand — APPROVED** [review]

**Comment by zekun000** [comment]
someone probably needs to prioritize fixing the transaction generator, otherwise whoever update the script functions needs to manually update

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 1 above)
Comment (wrwg) [reply aptos-move/framework/aptos-framework/sources/account.move:94]: Merge conflict resolution problem ... Resolved

**Comment by wrwg** [comment]
> someone probably needs to prioritize fixing the transaction generator, otherwise whoever update the script functions needs to manually update

Yes, I'll look. But likely we can live also until GTM with hand editing. @davidiw wants to get completely rid of the generator.

File: aptos-move/transaction-builder-generator/src/rust.rs [hunk 2]
```diff
@@ -217,6 +217,9 @@ impl ScriptFunctionCall {
             .with_comments(comments)
             .with_external_definitions(external_definitions)
             .with_serialization(!self.local_types);
+        for (k, v) in script_registry.iter() {
+            eprintln!("reg {}: {:?}", k, v);
+        }
```
Comment (davidiw) [inline aptos-move/transaction-builder-generator/src/rust.rs:222]: 😭

Comment (wrwg) [reply aptos-move/transaction-builder-generator/src/rust.rs:222]: Yeah, that code is not longer called, but better to remove this. Done.

File: ecosystem/typescript/sdk/src/token_client.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@ export class TokenClient {
   ): Promise<Types.HexEncodedBytes> {
     const payload: Types.TransactionPayload = {
       type: "script_function_payload",
-      function: "0x1::TokenTransfers::offer_script",
+      function: "0x1::tokenTransfers::offer_script",
```
Comment (geekflyer) [inline ecosystem/typescript/sdk/src/token_client.ts:122]:
```suggestion
      function: "0x1::token_transfers::offer_script",
```

**Pull request merged**